platform_secrets/service.py

```diff
@@ -24,6 +25,10 @@
 NAMESPACE_PROJECT_LABEL = "platform.apolo.us/project"
 
 
+def generate_namespace_name(org_name: str, project_name: str) -> str:
+    return f"platform--{org_name}--{project_name}"
```

Comment:
this combination can be larger than 253 characters. I think we will need to concatenate substring of the full name with the substring of the full name hash to avoid name conflicts

Reply:
@zubenkoivan pushed a new version of that function. Please check